account_cutoff_start_end_dates/__init__.py

```diff
@@ -1 +1,45 @@
 from . import models
+
+from openupgradelib import openupgrade
+from psycopg2 import sql
+
+
+def module_migration(cr):
+    if openupgrade.is_module_installed(cr, "account_cutoff_start_end_dates"):
```

Comment:
When the pre-init hook is executed, the module is never already installed because it is only executed when the module state is `to install`, see: https://github.com/odoo/odoo/blob/44624f5d51a266c4fc37644d3fc36b810e722ee4/odoo/modules/loading.py#L185

Reply:
correct, I'll modify it. thanks